documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -785,6 +785,12 @@ Hibernate Validator!
                 Pessoa Fídsica)|Brazil|None
 |`@TituloEleitoral`|`CharSequence`|Checks that the annotated character sequence represents
                 a Brazilian voter ID card number (link:$$http://ghiorzi.org/cgcancpf.htm$$[Título Eleitoral])|Brazil|None
+|`@NIP`|`CharSequence`|Checks that the annotated character sequence represents a Polish VAT identification
+ 				number ((link:$$https://pl.wikipedia.org/wiki/NIP$$[NIP]))|Poland|None
```

Comment:
We prefer using the short version for links now: `http://your.link/[your text]`. We fix them when we see them so you might as well clean up the line above about NIP. Thanks! (No need to fix all the links in the file, we do it when we update specific parts)